js/dropdown/shared/Multiselect.svelte

```diff
@@ -259,6 +260,11 @@
 					bind:value={input_text}
 					bind:this={filter_input}
 					on:keydown={handle_key_down}
+					on:keyup={(e) =>
+						dispatch("key_up", {
+							key: e.key,
+							input_value: input_text
+						})}
```

Comment:
I would just use keypress here, the issue with `keyup` is that it won't fire if you hold the key _down_ even though the text will be displayed which isn't very intuitive.